### ISSUE TITLE
Enhancement: Set max file size for kernel logger

### ISSUE THREAD

**Issue by stanmoore1** [kokkos/kokkos-tools#39]
I need to run the Kokkos kernel logger on hundred of thousands of MPI ranks for many hours to debug an issue. I only care about the last few kernels, i.e. when the code stops. It would be nice if there was an option for the kernel logger to overwrite (i.e. start at the beginning) of the output files if they get bigger than some threshold. Otherwise too much data could be generated.

**Comment by crtrott** [comment]
The right way to resolve this is to connect this with the kernel filter, and extent the kernel filter either with allowing a time start/stop threshold - or number of calls.

**Comment by stanmoore1** [comment]
@crtrott I don't think that will work. I have no idea when the code will stop, so I can't set a time limit or number of calls.
